src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -64,7 +68,8 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * Entry point for portlet omero-client-portlet. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
+ * Entry point for portlet omero-client-portlet. This class derives from {@link QBiCPortletUI},
+ * which is found in the {@code portal-utils-lib} library.
  *
```

Comment:
NIT: maybe add how and when this class should be used and add author and since to be consistent in our products.

Reply:
> Entry point for portlet omero-client-portlet.

Reply:
Adapted the documentation in a new commit to make streamline it with our current standard. Good point 🥇